lib/xebow/animation/static.ex

```diff
@@ -0,0 +1,114 @@
+defmodule Xebow.Animation.Static do
```

Comment:
I noticed a regression in the red and green static animations. When keys 12 + 1 or 12 + 3 are double tapped quickly to trigger the static animations, they get latched in the on state. I haven't traced the code on this one yet so I'm starting this convo on the module so it's easier to discuss than in the PR's main comment section.

Reply:
Indeed. Good catch.

I added the option to make `play_animation/1` synchronous and made the keys server use that option!

Reply:
Awesome, thanks!